packages/components/organisms/f-registration/src/components/Registration.vue

```diff
@@ -34,7 +34,6 @@
                 <form-field
                     v-model="firstName"
                     name="firstName"
-                    data-test-id="input-first-name"
```

Comment:
These can be added again if necessary but there are data-test-ids inside the form-field component which we can use to access the input element directly rather than its parent.